garnett/trajectory.py

```diff
@@ -1128,14 +1128,7 @@ def _regularize_box(positions, velocities,
             for i in range(angmom.shape[0]):
                 angmom[i, :] = rowan.multiply(quat, angmom[i, :])
```

Comment:
Is the `for` loop necessary here? I believe rowan can handle the element-wise multiplication, is that correct?

Reply:
Same applies to the orientations array above.

Reply:
Yup that's right, I'll fix.